src/ConnectCore.js

```diff
@@ -137,14 +137,15 @@ class ConnectCore {
   }
 
   // TODO support contract.new (maybe?)
-  contract (abi, uriHandler = this.uriHandler) {
-    const self = this
-    const txObjectHandler = (methodTxObject) => self.txObjectHandler(methodTxObject, uriHandler)
+  contract (abi) {
```

Comment:
I think this should still have the uriHandler passed in here or am I misunderstanding something? I'm pretty sure I'm using it somewhere. I'll double check.